scripts/cluster-setup.sh

```
@@ -24,9 +24,27 @@ PLUGIN_NAME=$(node -p "require('${REPO_DIR}/package.json').consolePlugin.name")
 	--console-plugin "${PLUGIN_NAME}=http://${HOST}:${PLUGIN_PORT}"
 
+log "installing kuadrant addon..."
+MAX_RETRIES=3
+RETRY_DELAY=10
+
+for attempt in $(seq 1 $MAX_RETRIES); do
+  if oinc addon install kuadrant@latest; then
+    log "Kuadrant addon ready"
+    break
+  fi
+  if [ $attempt -lt $MAX_RETRIES ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)